pandas/plotting/_matplotlib/misc.py

```diff
@@ -130,7 +146,7 @@ def normalize(series):
     if ax is None:
         ax = plt.gca(xlim=[-1, 1], ylim=[-1, 1])
 
-    to_plot = {}
+    to_plot: Dict[Label, List[List[Any]]] = {}
```

Comment:
Any? from matplotlib scatter, x, y : float or array-like, shape (n, )

vendoring https://github.com/predictive-analytics-lab/data-science-types/blob/master/matplotlib-stubs/axes.pyi might help.

maybe just Dict[Label, List[List]] while we don't enforce adding type parameters and we can revisit this later, rather than adding Any which maybe harder to track.